_includes/pq-compression/makes-a-cookbook.mdx

```diff
@@ -0,0 +1 @@
+PQ relies on a codebook to compress the original vectors. The codebook defines 'centroids' that are used to calculate the compressed vector. Weaviate’s PQ implementation uses existing data to train the codebook. You must have some vectors loaded before you enable PQ so Weaviate can use them to define the centroids. You should have 10,000 to 100,000 vectors loaded before you enable PQ.
```

Comment:
Why are "centroids" in quotes? Are they not actually centroids or some approximate value?